src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,97 @@
+package duke.storage;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import duke.exception.FileUpdateFailException;
+import duke.exception.InvalidFileFormatException;
+import duke.task.Task;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+
+/**
+ * Handles the interactions with the user's CSV file.
+ * Includes creating, updating file and retrieving list of tasks from CSV file.
+ */
+public class Storage {
+
+    private final String dataDir;
```

Comment:
Try not to use shortened names for variables so that the variable name makes more sense

Reply:
Okay good point. Made the relevant changes.